.github/workflows/robot/internal/bot/backport.go

```diff
@@ -120,11 +98,23 @@ func (b *Bot) Backport(ctx context.Context) error {
 			Link: url.URL{
 				Scheme: "https",
 				Host:   "github.com",
-				Path:   path.Join(b.c.Environment.Organization, b.c.Environment.Repository, "pull", strconv.Itoa(number)),
+				// Both base and head are safe to put into the URL: base has
+				// had the "branchPattern" regexp run against it and head is
+				// formed from base so an attacker can not control the path.
+				Path: path.Join(b.c.Environment.Organization, b.c.Environment.Repository, "compare", fmt.Sprintf("%v...%v", base, head)),
+				RawQuery: url.Values{
+					"expand": []string{"1"},
+					"title":  []string{fmt.Sprintf("[%v] %v", strings.Trim(base, "branch/"), pull.UnsafeTitle)},
+					"body":   []string{fmt.Sprintf("Backport #%v to %v", b.c.Environment.Number, base)},
+				}.Encode(),
```

Comment:
`labels=backport` perhaps?

Reply:
Labels are automatically assigned upon creation:

https://github.com/gravitational/teleport/blob/master/.github/workflows/label.yaml
https://github.com/gravitational/teleport/blob/master/.github/workflows/robot/internal/bot/label.go#L53-L57

Reply:
That prefix check should be done on `pull_request.base.ref`, not `pull_request.head.ref` - it's not actually working to label backports at the moment.